conda/environments/cudf_dev_cuda9.2.yml

```diff
@@ -10,11 +10,10 @@ dependencies:
 - rmm=0.9.*
 - cmake>=3.12
 - python>=3.6,<3.8
-- numba>=0.41
-- pandas>=0.23.4
-- pyarrow=0.12.1
+- numba>=0.41,<0.45
+- pandas>=0.24.2,<0.25
```

Comment:
Why not support Pandas 0.25.0?

Reply:
Follow-up PR where Pandas 0.25 currently breaks us. Didn't want to add even more to this PR 😅

Reply:
`pandas.compat.string_types` ftw (missing in 0.25).

Reply:
I already fixed it! Now I have to unfix it. :p